albumentations/augmentations/domain_adaptation.py

```diff
@@ -80,21 +111,21 @@ class HistogramMatching(ImageOnlyTransform):
         uint8, uint16, float32
     """
 
-    def __init__(self, reference_images: List[str], blend_ratio=(0.5, 1.0), read_fn=cv2.imread, p=0.5):
-        super().__init__(p=p)
+    def __init__(
+        self,
+        reference_images: List[Union[str, np.ndarray]],
+        blend_ratio=(0.5, 1.0),
+        read_fn=read_bgr_image,
```

Comment:
Maybe use `read_rgb_image` as a default `read_fn` both for `HistogramMatching` and `FDA`? I think most users are working with RGB images, and this will be a better default.

Reply:
done